src/test/java/org/apache/sysds/test/functions/privacy/MatrixMultiplicationPropagationTest.java

```diff
@@ -46,26 +47,36 @@ public void setUp() {
 	}
 
 	@Test
-	public void testMatrixMultiplicationPropagation() throws JSONException {
-		matrixMultiplicationPropagation(true, true);
+	public void testMatrixMultiplicationPropagationPrivate() throws JSONException {
```

Comment:
I see the JSON throwing is here as well.

Reply:
Yes, that is on purpose. It throws a JSONException if the privacy constraints are not written to the result metadata file.